Signal/src/AppDelegate.m

```diff
@@ -317,7 +317,13 @@ - (void)application:(UIApplication *)application
  */
 - (BOOL)application:(UIApplication *)application continueUserActivity:(nonnull NSUserActivity *)userActivity restorationHandler:(nonnull void (^)(NSArray * _Nullable))restorationHandler
 {
-    DDLogWarn(@"%@ called %s with userActivity: %@, but not yet supported.", self.tag, __PRETTY_FUNCTION__, userActivity);
+    if ([userActivity.activityType isEqualToString:@"INStartVideoCallIntent"]) {
+        [[Environment getCurrent].callService handleCallKitStartVideo];
```

Comment:
I'm only a little surprised.

Just to keep in mind the comment below, we'll eventually also use this method to initiate a CallKit voice call when someone chooses to initiate a Signal call from their contacts, or from the "callback" action on the system call screen when a call fails.